packages/create-cloudflare/e2e/helpers/framework-helpers.ts

```diff
@@ -283,9 +283,9 @@ export async function verifyPreviewScript(
 
 	try {
 		// Some frameworks take quite a long time to build the application (e.g. Docusaurus)
-		// so wait up to 5 mins for the dev-server to be ready.
+		// so wait up to 3 mins for the dev-server to be ready.
```

Comment:
why this change? you didn't modify this time, did you?

Reply:
I did, in the next line